teams/wg-rustc-dev-guide.toml

```
@@ -4,7 +4,7 @@ kind = "working-group"
 
 [people]
 leads = ["spastorino", "jyn514"]
-members = ["spastorino", "amanjeev", "tshepang", "togiberlin", "JohnTitor", "chrissimpkins", "LeSeulArtichaut", "jyn514"]
+members = ["spastorino", "amanjeev", "tshepang", "togiberlin", "JohnTitor", "chrissimpkins", "LeSeulArtichaut", "jyn514", "Camelid"]
```

I think you need to lowercase my name to match how it is in my `people/` file:

```suggestion
members = ["spastorino", "amanjeev", "tshepang", "togiberlin", "JohnTitor", "chrissimpkins", "LeSeulArtichaut", "jyn514", "camelid"]
```